crates/differ/src/apply_diff.rs

```diff
@@ -66,5 +74,10 @@ mod tests {
 			super::apply_diff("Saturday", test_vec_2)
 		);
 		assert_eq!(String::from("SETS"), super::apply_diff("RESET", test_vec_3));
+		assert_eq!(String::from("heeoy"), super::apply_diff("hey", test_vec_4));
+		assert_eq!(
+			String::from("skater"),
+			super::apply_diff("kate", test_vec_5)
+		);
```

Comment:
In a future commit, can we make it so these assert_eq! statements are more consistent in style? Thanks!